core/index.ts

```diff
@@ -21,3 +21,4 @@ export * from './post.js'
 export * from './i18n.js'
 export * from './html.js'
 export * from './slow.js'
+export { getTestEnvironment } from './test/utils.js'
```

Comment:
Let’s move it to `./environment.ts` then

Reply:
Similar question for `./dom-parser.ts`

Is it ok to move it to `core` and to export it from there if it's needed?

We need it for loader-tests project and I believe in many more places soon.

Reply:
No, it is DOM polyfills hack. Let’s do it again in every project (I’m afraid to export polyfills since it could break environments by accidentally change globals).

Reply:
Deal :+1: